disgames/mixins/hangman.py

```diff
@@ -28,11 +29,28 @@ def _show_guesses(self, embed, guesses):
 
     @commands.command("hangman", aliases=["hm"])
     async def command(self, ctx: commands.Context):
-        h = self.make_hangman.__globals__['__file__'].split('\\')
-        h[-1] = 'words.txt'
-        with open('\\'.join(h), "r") as dictionary:
-            words = [s.lower() for s in dictionary.read().splitlines() if len(s) >= 4 and len(s) <= 6]
-        word = list(random.choice(words))
+        try:
+            with open('./words.txt', 'r') as file:
+                data = file.read().splitlines()
+                data = random.choice(data)                
+                words = data
+        except:
+            words = []
+            target_url = 'https://raw.githubusercontent.com/andrewthederp/Disgames/main/disgames/mixins/words.txt'
+            async with aiohttp.ClientSession() as session:
+                async with session.get(target_url) as resp:
+                    data = await resp.text()
+                with open('./words.txt', 'w') as file:
+                    file.write(data)
+            with open('./words.txt', 'r') as file:
+                data = file.read().splitlines()
+                data = random.choice(data)
+                words = data
+                
+        words = str(words).replace('\n', '')
+                
+        word = list(words)
```

Comment:
These could be split into a different function which *could* be named something like `get_words` or something

Reply:
Do I have to do this? Or suggestion?

Reply:
a suggestion, we could change it in the future before another release..

Reply:
Alr

Reply:
Resolved in https://github.com/Jerry-py/Disgames/commit/0576337f4c04e4cb6f49d4c724b475f3d9c84fd5